app/assets/javascripts/discourse/app/components/post-stream.gjs

```diff
@@ -355,6 +393,21 @@ export default class PostStream extends Component {
         {{/let}}
       {{/each}}
 
+      {{#if @postStream.canAppendMore}}
```

Comment:
Same as above.

Does it make sense to move the region to inside the `if` above the load more component?